R/readNGWMNdata.R

```diff
@@ -119,19 +118,19 @@ readNGWMNdata <- function(service, ..., asDateTime = TRUE, tz = "UTC"){
 #' \donttest{
 #' #one site
 #' site <- "USGS.430427089284901"
-#' oneSite <- readNGWMNlevels(siteNumbers = site)
+#' #oneSite <- readNGWMNlevels(siteNumbers = site)
 #' 
 #' #multiple sites
 #' sites <- c("USGS:272838082142201","USGS:404159100494601", "USGS:401216080362703")
-#' multiSiteData <- readNGWMNlevels(sites)
```

Comment:
are these not working again?

Reply:
They are taking forever....I saw in slack they were having some hiccups again.

Reply:
Maybe (in the future) it would be worth building in a `verbose` argument that if true tells you when different steps of the call finish?  I had built in at one point a message when the file was downloaded so you knew when something taking forever was due to the download time or parsing issues.

Reply:
there's a verbose option via httr...but yeah, we could add on to that. Want to make an issue?

Reply:
sure